docs/unit-testing-for-monitoring-metrics.md

```diff
@@ -10,43 +10,38 @@ The ARO monitor component (the part of the aro binary you activate when you exec
 
 ![Aro Monitor Architecture](img/AROMonitor.png "Aro Monitor Architecture")
 
-To send data to Geneva the monitor uses an instance of a Geneva MDM container as a proxy of the Geneva API. The MDM container accepts statsd formatted data (the Azure Geneva version of statsd, that is) over a UNIX (Domain) socket. The MDM container then forwards the metric data over a https link to the Geneva API. Please note that using a Unix socket can only be accessed from the same machine. 
+To send data to Geneva the monitor uses an instance of a Geneva MDM container as a proxy of the Geneva API. The MDM container accepts statsd formatted data (the Azure Geneva version of statsd, that is) over a UNIX (Domain) socket. The MDM container then forwards the metric data over a https link to the Geneva API. Please note that a Unix socket can only be accessed from the same machine. 
 
 The monitor picks the required information about which clusters should actually monitor from its corresponding Cosmos DB. If multiple monitor instances run in parallel  (i.e. connect to the same database instance) as is the case in production, they negotiate which instance monitors what cluster (see : [monitoring.md](./monitoring.md)). 
 
 
-
-
-## Unit Testing Setup
+# Unit Testing Setup
 
 If you work on monitor metrics in local dev mode (RP_MODE=Development) you most likely want to see your data somewhere in Geneva INT (https://jarvis-west-int.cloudapp.net/) before you ship your code.
 
-There are two ways to set to acchieve this: 
-- Run the Geneva MDM container locally (won't work on macOS, see Remote Container section below)
+There are two ways to set to achieve this: 
+- Run the Geneva MDM container locally
 - Spawn a VM, start the Geneva container there and connect/tunnel to it.
 
-### Local Container Setup
+and two protocols to chose from:
+- Unix Domain Sockets, which is the way production is currently (April 2022) run
+- or UDP, which is much easier to use and is the way it will be used on kubernetes clusters in the future
+
+## Local Container Setup
 
 Before you start, make sure :
 - to run `source ./env`
 - you ran `SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets` before 
 - know which "account" and "namespace" value you want to use on Geneva INT for your metric data and
   update your env to set the 
   - CLUSTER_MDM_ACCOUNT
-  - CLUSTER_MDM_NAMESPACE
-   
-
-  variables before you start the monitor. 
+  - CLUSTER_MDM_NAMESPACE variables before you start the monitor. 
 
-An example docker command to start the container locally is here (you may need to adapt some parameters):
-[Example](../hack/local-monitor-testing/sample/dockerStartCommand.sh). The script will configure the mdm container to connect to Geneva INT
+ The container needs to be provided with the Geneva key and certificate. For the INT instance that is the rp-metrics-int.pem you find in the secrets folder after running the `make secrets` command above.  
```

Comment:
I don't believe this key/cert exist yet in the new subscription. Do you know what needs to be done to create/copy them?

Reply:
I used the ones from the old subscription, which is not an option anymore. If they are missing in the new storage account we'll need to get them added.

I effectively reverted the documentation to the state before the UDP option because MSFT introduced new security rules that block most of the local testing. Will need to look at that some other time in another PR.